src/OrchardCore.Modules/OrchardCore.Demo/Module.txt

```diff
@@ -6,3 +6,9 @@ Version: 2.0.x
 OrchardVersion: 2.0.x
 Description: Test
 Category: Samples
+Features:
+    OrchardCore.Demo.Users.EntityFrameworkCore:
+        Name: Users Demo (Entity Framework Store)
+        Description: Use Entity Framework Core ORM in order to store User data
+        Category: Samples
+        Dependencies: OrchardCore.Users.EntityFrameworkCore
```

Comment:
For infos

A module can have only one default feature but as soon that you have a `Features:` section, this default feature definition is ignored. So, here we need to define it as a 1st feature in this section.

Note: When not defined in a feature, `Priority, Category, Description` are grabbed from the module.

So, here we would need, e.g.

    Name: Orchard Demo
    AntiForgery: enabled
    Author: The Orchard Team
    Website: http://orchardproject.net
    Version: 2.0.x
    OrchardVersion: 2.0.x
    Features:
        OrchardCore.Demo:
            Name: Orchard Demo
            Description: Test
            Category: Samples
        OrchardCore.Demo.Users.EntityFrameworkCore:
            Name: Users Demo (Entity Framework Store)
            Description: Use Entity Framework Core ORM in order to store User data
            Category: Samples
            Dependencies: OrchardCore.Users.EntityFrameworkCore

Reply:
Thank you for review and explanation, I'm fixing it right now